.github/pull_request_template.md

```diff
@@ -28,6 +28,7 @@ More information about contributing to NAV can be found in the
 * [ ] The first line of the commit message continues the sentence "If applied, this commit will ...", starts with a capital letter, does not end with punctuation and is 50 characters or less long. See https://cbea.ms/git-commit/
 * [ ] This pull request is based on the correct upstream branch: For a patch/bugfix affecting the latest stable version, it should be based on that version's branch (`<major>.<minor>.x`). For a new feature or other additions, it should be based on `master`.
 * [ ] If applicable: Created new issues if this PR does not fix the issue completely/there is further work to be done
+* [ ] If it's not obvious from a linked issue, described how to interact with NAV in order for a reviewer to observe the effects of this change first-hand (commands, URLs, UI interactions)
```

Comment:
Haha, there was a typo in my own suggestion...

```suggestion
* [ ] If it's not obvious from a linked issue, describe how to interact with NAV in order for a reviewer to observe the effects of this change first-hand (commands, URLs, UI interactions)
```

Reply:
No, this is actually my change that I introduced - it's just that all the checks are in past tense to basically say, when the person is filling it out "I did this thing - check"

Reply:
Out of scope: the tenses of the checks are not consistent